Allow correct gender translations for course unit prev/next buttons

OmarIthawi opened the pull request:
A translator of the Romanian language complained on Transifex about issues that both [Previous](https://www.transifex.com/open-edx/edx-platform/translate/#ro/mako/14756962?q=text%3Aprevious) (unit) and [Next](https://www.transifex.com/open-edx/edx-platform/translate/#ro/mako/14756963?q=text%3Anext) have no context that translators can use to provide accurate gendered translation:

> #### [Issue on Transifex](https://www.transifex.com/open-edx/edx-platform/translate/#ro/mako/14756963)
> ` @itsalexcoman`: The gender of the noun to which this adjective refers is crucial for a proper Romanian translation.


### TODO
 - [x] I have checked the a the "Next" buttons would show the next ["Unit" (aka vertical)](https://edx.readthedocs.io/projects/edx-open-learning-xml/en/latest/organizing-course/course-xml-file.html#course-verticals). Please double check that for me, since the terminology can be confusing even for those who are familiar with Open edX. 
 - [x] Since `gettext` has no awareness of gender, I've used the context-aware function `pgettext` to provide more context for the translators. 
 - [x] A `Translators` comment have been added.
 - [x] Test in my devstack (Hawthorn via clean cherry-pick) that the UI is still functional:
        <kbd>![image](https://user-images.githubusercontent.com/645156/63652666-69162d80-c76b-11e9-9206-ecfee96d24e3.png)</kbd>

 - [x] Ensure strings are being extracted properly with context
       <kbd>![image](https://user-images.githubusercontent.com/645156/63652518-bbeee580-c769-11e9-9329-d4eb4162ca0c.png)</kbd>

 - [x] Jenkins tests pass -- I suppose no new tests should be added.
 - [ ] Once merged, get back to `@itsalexcoman` about the issue.

cc: @nedbat

edx-webhook commented:
Thanks for the pull request, @OmarIthawi! I've created [OSPR-3803](https://openedx.atlassian.net/browse/OSPR-3803) to keep track of it in JIRA. JIRA is a place for product owners to prioritize feature reviews by the engineering development teams.

Feel free to add as much of the following information to the ticket:
- supporting documentation
- edx-code email threads
- timeline information ("this must be merged by XX date", and why that is)
- partner information ("this is a course on edx.org")
- any other information that can help Product understand the context for the PR

All technical communication about the code itself will still be done via the GitHub pull request interface. As a reminder, [our process documentation is here](http://edx-developer-guide.readthedocs.org/en/latest/process/overview.html).

OmarIthawi commented:
jenkins rerun bokchoy

nedbat commented on lms/templates/seq_module.html:
```diff
@@ -88,10 +90,12 @@
     <button class="sequence-nav-button button-previous">
       <span class="icon fa fa-chevron-prev" aria-hidden="true"></span>
-      <span>${_('Previous')}</span>
+      ## Translators: A button for showing the Next Unit
```
s/Next/Previous/

nedbat commented on lms/templates/seq_module.html:
```diff
@@ -88,10 +90,12 @@
     <button class="sequence-nav-button button-previous">
       <span class="icon fa fa-chevron-prev" aria-hidden="true"></span>
-      <span>${_('Previous')}</span>
+      ## Translators: A button for showing the Next Unit
```
s/Next/Previous/

OmarIthawi replied:
Done

OmarIthawi commented:
Thanks for the review @nedbat.

I'm done with this PR, however I can _not_ test it on my `master` devstack, as I'm having a weird issue. I've cherry-picked it on my `hawthorn` devstack and it looks good.

I've also addressed your comment. Please take another look.

edx-webhook commented:
@OmarIthawi 🎉 Your pull request was merged!

Please take a moment to [answer a two question survey](https://docs.google.com/forms/d/e/1FAIpQLSceJOyGJ6JOzfy6lyR3T7EW_71OWUnNQXp68Fymsk3MkNoSDg/viewform?usp=pp_url&entry.1671973413=edx/edx-platform&entry.867055334=https://github.com/edx/edx-platform/pull/21440&entry.1484655318=https://github.com/OmarIthawi&entry.752974735=2019-08-25+10:29&entry.1917517419=2019-08-25+21:16&entry.2133058324=Yes) so we can improve your experience in the future.

OmarIthawi commented:
Thanks @nedbat!
